code/game/objects/items.dm

```diff
@@ -7,181 +7,111 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item
 	name = "item"
 	icon = 'icons/obj/items_and_weapons.dmi'
-
-	/// The icon state for the icons that appear in the players hand while holding it. Gotten from /client/var/lefthand_file and /client/var/righthand_file
```

Comment:
Why'd you delete all the comments in this file?

Reply:
Mistake, will fix later